### PR TITLE
Docker Compose for whole system

### PR DESCRIPTION
**Pull request by seanmcilvenna**
* docker-compose.yml that has all services needed to run the entire stack of services
* supporting services include:
  * kafka broker
  * kafka ui
  * sql server
  * mongo
  * postgres
  * redis
  * loki
  * prometheus
  * grafana
* init containers;
  * broker has init that creates topics
  * sql database has a `create_dbs.sql` file that creates the dbs
* run `docker compose build` from root to build all services
* run `docker compose up` to run all services
* each service declares dependencies
* occasional modifications to default app settings
* a new appsettings.Docker.json file that is used by the docker-compose to launch the services with a specific port (on http, not https)
* modified (and created) .dockerignore files to exclude built artifacts (new .dockerignores needed at the root context of each build)
* some changes made to the Java services to have them build and run successfully in the docker compose
* consistency across all projects for naming conventions like the name of the app in logs "link-cloud"
* all services aggregate logs to loki and can be accessed via grafana
* deleted project folders that are no longer used and no longer relevant

### CODE REVIEW

**Review by edward-miller-lcg — APPROVED** [review]
Would like to see @dvargaslantana or @amphillipsLGC or @smailliwcs provide an approval as well in case I missed something. But overall looks good to me.

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+LINK_DB_USER=link-db-user
+LINK_DB_PASS=link-db-pass
+REDIS_PASS=suT1ChwYtkwWfbV1bUfRvQZO1YNhNRMj
```
Comment (smailliwcs) [inline .env:3]: Is this sensitive?

**Review by smailliwcs — APPROVED** [review]
Approved, and tacked on further Java-specific changes in a `java-deployment` branch.

File: .env (hunk 1 above)
Comment (seanmcilvenna) [reply .env:3]: No. This is a new uniquely generated value just for this docker-compose.

**Pull request merged**